frontend/src/components/ComboboxInput.vue

```diff
@@ -118,31 +118,31 @@ const onInput = async () => {
   await showUpdatedDropdown()
 }
 
-const keyup = () => {
-  if (focusedItemIndex.value > 1) {
+const HTMLNodes = computed(() =>
```

Comment:
As they are all child nodes from dropdownItemsRef, I renamed them to dropdownItems, later to be replaced by on common list (instead of filtered nodes)